samples/subscribing-to-streams/Program.cs

```diff
@@ -29,21 +30,21 @@ class Program {
 			#region subscribe-to-stream-from-position
 			await client.SubscribeToStreamAsync(
 				"some-stream",
-				StreamPosition.FromInt64(20),
+				SubscriptionStreamPosition.After(StreamPosition.FromInt64(20)),
 				EventAppeared);
 			#endregion subscribe-to-stream-from-position
 		
 			#region subscribe-to-stream-live
 			await client.SubscribeToStreamAsync(
 				"some-stream",
-				StreamPosition.End,
+				SubscriptionStreamPosition.Live,
```

Comment:
We should actually call this `SubscriptionStreamPosition.End` rather than `Live` so that it stays consistent with the other gRPC clients